### PR TITLE
Add upload metadata schema in JSON

### PR DESCRIPTION
**Pull request by sven1103**
Provides a first draft of a upload metadata schema, that describes the current metadata object that is used in dropboxes such as `register-imgag-dropbox`.

### CODE REVIEW

**Comment by sven1103** [comment]
[Add metadata json to metadata model](https://trello.com/c/N6FoEgTU/3-add-metadata-json-to-metadata-model)

File: upload-metadata.schema.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                }
+            }
+        },
+        "sample": {
+            "type": "object",
+            "properties": {
+                "genome": {
+                    "type": "string",
+                    "examples": ["GRCh37"]
+                },
```
Comment (KochTobi) [inline upload-metadata.schema.json]: This is only for genomics?

File: upload-metadata.schema.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                "id_genetics": {
+                    "type": "string",
+                    "examples": ["GS000000_01"]
+                },
```
Comment (KochTobi) [inline upload-metadata.schema.json]: maybe add a `"description" : ` here. I do not understand what `id_genetics` contains. What sort of id? Where does it come from? Can it be multiple ids from different sources?

File: upload-metadata.schema.json (hunk 1 above)
Comment (sven1103) [reply upload-metadata.schema.json]: so far yes. It is not a new model, it is just a description of the current one used in the IMGAG dropbox for documentation purposes only.

**Review by KochTobi — APPROVED** [review]

**Pull request merged**